indico/modules/events/templates/display/_manage_button.html

```diff
@@ -0,0 +1,132 @@
+{% from 'events/notes/_note.html' import render_toggle_button %}
+
+{% macro _render_note_editor(item) -%}
+    <li>
+        <a href=""
+           class="notes-editor"
+           data-href="{{ url_for('event_notes.edit', item) }}"
+           data-title="{% trans %}Edit minutes{% endtrans %}">
```

Comment:
shouldn't this also use add/edit depending on whether there is already a note? could abuse a block for this:

```
data-title="{{ self.edit_minutes_title() }}">
    {% block edit_minutes_title %}
        {% if item.has_note -%}
            {% trans %}Edit minutes{% endtrans %}
        {%- else -%}
            {% trans %}Add minutes{% endtrans %}
        {%- endif %}
    {% endblock %}
```